src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -102,6 +104,9 @@ public SearchesClient getSearchesClient() {
         return searchesClient;
     }
 
+    @Override
+    public ServerSessionClient getServerSessionClient() { return serverSessionClient; }
```

Comment:
Server session is deprecated so we didn't add it to the repository client. I think all of its methods are essentially noops.